### DISCUSSION TITLE
Where do I find the options for .props()

### DISCUSSION THREAD

**Discussion by HerzMichi** [zauberzeug/nicegui#1415] [Q&A] [answered]
### Question

For example:
```python
from nicegui import ui

radio1 = ui.radio([1, 2, 3], value=1).props('inline')
radio2 = ui.radio({1: 'A', 2: 'B', 3: 'C'}).props('inline').bind_value(radio1, 'value')

ui.run()
```

.props("inline") sets the radio buttons horizontally. But what are all the other options that I could use?

Thank you in advance :)

**Comment by falkoschindler** [comment]
NiceGUI's `ui.radio` uses Quasar's [Radio](https://nicegui.io/documentation/radio) component. There you can find a good overview of available props.
Apparently our documentation is missing some links to Quasar. We'll fix that.

**Reply by HerzMichi** [reply]
Maybe I can ask more specifically. 
`ui.toggle([1, 2, 3], value=1)`
How do I make this toggle component vertical. I guess instead of `.props('inline')`  something like `.props('vertical')`. But I can not find the proper way to do it in the Quasar documentation.

**Reply by HerzMichi** [reply]
This works with `ui.radio()` but not with a `ui.toggle()`, but thank you.

**Reply by falkoschindler** [reply, marked as answer]
No, Quasar has no "vertical" prop for `ui.toggle`.
